crates/teloxide-core/schema.ron

```diff
@@ -1287,6 +1287,11 @@ Schema(
             tg_doc: "https://core.telegram.org/bots/api#sendpaidmedia",
```

Comment:
` to channel chats` in the method description should be missing after 7.9 update